var/spack/repos/builtin/packages/paraview/package.py

```diff
@@ -73,6 +73,7 @@ class Paraview(CMakePackage, CudaPackage):
     variant("pagosa", default=False, description="Build the pagosa adaptor")
     variant("eyedomelighting", default=False, description="Enable Eye Dome Lighting feature")
     variant("adios2", default=False, description="Enable ADIOS2 support", when="@5.8:")
+    variant("openpmd", default=False, description="Enable openPMD support (w/ ADIOS2/HDF5)")
```

Comment:
FWIW. You should be able to add `when="@:5.8 +adios2+hdf5",` here and the variant will only be allowed for a spec that satisfies those constraints.  This would allow you to get rid of the conflicts on lines 109 and 111.

Reply:
See https://spack.readthedocs.io/en/latest/packaging_guide.html#conditional-variants.

Reply:
Cool, thanks! :)

Reply:
Updated :)